framework/src/modules/interoperability/utils.ts

```diff
@@ -130,6 +131,9 @@ export const isInboxUpdateEmpty = (inboxUpdate: InboxUpdate) =>
 	inboxUpdate.outboxRootWitness.siblingHashes.length === 0 &&
 	inboxUpdate.outboxRootWitness.bitmap.length === 0;
 
+export const isOutboxRootWitnessEmpty = (outboxRootWitness: OutboxRootWitness) =>
+	outboxRootWitness.siblingHashes.length === 0 || outboxRootWitness.bitmap.length === 0;
```

Comment:
Thinking if this should be a `&&` or `||` 🤔

Reply:
double checked with research team and they confirmed that we can say if any of the property is empty we can consider outboxRootWitness to be empty